website/content/docs/connect/observability/grafanadashboards/service-to-servicedashboard.mdx

```diff
@@ -0,0 +1,179 @@
+---
+layout: docs
+page_title: Dashboard for monitoring Consul service-to-service mesh
+description: >-
+  This documentation provides an overview of the Service to Service Dashboard.
+---
+
+# Service-to-service dashboard
+
+This page provides reference information about the Grafana dashboard configuration included in [this GitHub repository](https://github.com/hashicorp/consul/tree/main/grafana/service-to-servicedashboard.json). The service-to-service dashboard provides deep visibility into the traffic and interactions between services within the Consul service mesh. It focuses on critical metrics such as logs, error rates, traffic patterns, and success rates, all of which help operators maintain smooth and reliable service-to-service communication. 
```

Comment:
The link here is incorrect for github. I think it should be https://github.com/hashicorp/consul/blob/main/grafana/consulservicetoservicedashboard.json